_news-items/2025-08-06-riverside.html

```diff
@@ -0,0 +1,24 @@
+---
+# Copyright 2025 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
+title: Thank you Riverside Research, sponsor of the seL4 Summit 2025 reception
+anchor: sponsor25-riverside
+
+img: /Foundation/Membership/LOGOS/riverside-research.svg
+alt: Riverside Research Logo
+link: https://www.riversideresearch.org/
+---
+
+<p>
+  The seL4 Foundation thanks <a href="https://www.riversideresearch.org/">Riverside Research</a>
+  for sponsoring the <a href="../Summit/2025">seL4
+  Summit 2025</a> reception.
```

Comment:
I suggest adding:
"Riverside Research [recently acquired] Cog Systems, a [Founding Member] of the [seL4 Foundation]"
with [recently acquired]: https://www.riversideresearch.org/insights/riverside-research-acquires-cog-systems-to-strengthen-cyber-resilience-and-secure-virtualization-capabilities